### PR TITLE
[PATCH]-EDA-1285-next turn refactor

### PR DESCRIPTION
**Pull request by vgatica-eb**

### CODE REVIEW

File: game/game.py [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
 
-    def next_turn(self, players, current_player):
-        for player in players:
-            if player is current_player:
-                index = players.index(current_player)
-                current_player = players[(index + 1) % len(players)]
-                return (current_player)
+    def next_turn(self):
+        for player in self.players:
+            if player.name is self.current_turn:
```
Comment (nicolasnunez-eb) [inline game/game.py:16]: Why don't you use a reference to a player in current_turn instead of the name

File: game/game.py [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
+                index = self.players.index(player)
+                self.current_turn = self.players[(index + 1) % len(self.players)].name
+                return (self.current_turn)
```
Comment (nicolasnunez-eb) [inline game/game.py:19]: parenthesis are not necessary

**Review by srobledo-eb — APPROVED** [review]

File: game/game.py (hunk 1 above)
Comment (vgatica-eb) [reply game/game.py:16]: First of all. Thank u @nicolasnunez-eb to comment my PR!!! We were thinking about use a number, for example the index. But current_player is an attribute that we are going to send to server, and we need send it as a name. So, we decided manage it as a string and not a number reference

File: game/game.py (hunk 2 above)
Comment (vgatica-eb) [reply game/game.py:19]: fixing it

**Comment by vgatica-eb** [comment]
I was working with next_turn method and tests

**Comment by vgatica-eb** [comment]
I was working with next_turn and tests

**Review by arielaranda-eb — COMMENTED** [review]
✅  Great work!

**Pull request merged**